code/modules/mob/living/carbon/carbon.dm

```diff
@@ -106,6 +106,8 @@
 			Weaken(3)
 	if(override)
 		return override
+	else if(shock_damage > 400)

```

Comment:
>= isn't needed
We have a max health define or var somewhere, right? Just do MAX_HEALTH*2 or whatever